pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -317,6 +330,24 @@ public BuildTask getBuild(String identifier) {
     private boolean isBuildAlreadySubmitted(BuildTask buildTask) {
         return buildTasks.contains(buildTask);
     }
+    
+    /**
+     * Dummy wrapper for interface RunningEnvironment
+     * @author Jakub Bartecek <jbartece@redhat.com>
+     *
+     */
+    private class RunningEnvironmentWrapper {
+        
+        private RunningEnvironment runningEnvironment;
+        
+        public RunningEnvironment getRunningEnvironment() {
+            return runningEnvironment;
+        }
+        
+        public void setRunningEnvironment(RunningEnvironment runningEnvironment) {
+            this.runningEnvironment = runningEnvironment;
+        }
+    }
 
```

Comment:
You can use generic org.jboss.pnc.common.util.ObjectWrapper instead of specific RunningEnvironmentWrapper.